.travis.yml

```diff
@@ -7,33 +7,52 @@ matrix:
   include:
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=3.4.* PHPSTAN=true
+      env: SYMFONY_VERSION=^3.4 PHPSTAN=true
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=3.4.* PHP_CS_FIXER=true
+      env: SYMFONY_VERSION=^3.4 PHP_CS_FIXER=true
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=3.4.* UNIT_TESTS=true
+      env: SYMFONY_VERSION=^3.4 UNIT_TESTS=true
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=4.0.* UNIT_TESTS=true
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
     - php: 7.2
       sudo: false
-      env: SYMFONY_VERSION=4.0.* UNIT_TESTS=true
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
+    - php: 7.3
+      sudo: false
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
     - php: 7.1
       services: docker
       sudo: required
-      env: SYMFONY_VERSION=3.4.* FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+      env: SYMFONY_VERSION=^3.4 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
     - php: 7.1
       sudo: required
       services: docker
-      env: SYMFONY_VERSION=4.0.* FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+      env: SYMFONY_VERSION=^4.0 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+    - php: 7.2
+      services: docker
+      sudo: required
+      env: SYMFONY_VERSION=^3.4 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+    - php: 7.2
+      sudo: required
+      services: docker
+      env: SYMFONY_VERSION=^4.0 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+    - php: 7.3
```

Comment:
Php version does not matter for functional tests because they are run in the container which has php 7.1 inside it.